docs/core/platform/backup.md

```diff
@@ -2,11 +2,135 @@
 title: "Backup"
 ---
 
+There are two main purposes for an integration to implement a backup platform:
```

Comment:
```suggestion
Integrations may implement the backup platform for one of both of the following purposes:
```

Reply:
An integration can do both.

Reply:
I made a typo. I meant "for one **or** both".